docker-compose.yml

```diff
@@ -46,6 +46,7 @@ services:
       INFLUXURL: http://influxdb:8086
       NUOCMD_API_SERVER: nuoadmin1:8888
       NUOCD_HOSTNAME: sm
+      NUOCD_USE_NUOADMIN: "true"
```

Comment:
Not sure how this is still necessary. The non-nuoadmin code should just be removed from nuodb-collector completely and the behavior enabled by `NUOCD_USE_NUOADMIN=true` should be the only option.

Reply:
It's not obvious what NUOCD_USE_NUOADMIN does.  There are two different ways to find nuodb process.   The default was / is to use pgrep nuodb.   The other option is to periodically look for nuodb process on local machine via nuocmd get processes.   That is what this variable does.  I need to confirm.  I ended up merging a change that I did for Nik to eliminate the need to use shared process space between the collector and the engine containers.   I can roll back this change.